docs/pages/index.tsx

```diff
@@ -99,10 +99,10 @@ export default function LandingPage() {
             <Content maxWidth="md">
               <Logo src="/static/logo_raw.svg" alt="" />
               <div>
-                <Title variant="h3" component="h1" color="inherit" gutterBottom>
+                <Title variant="h3" component="h1" color="white" gutterBottom>
```

Comment:
```suggestion
                <Title variant="h3" component="h1" color={theme => theme.palette.mode === 'dark' ? '#fff' : 'primary.main'} gutterBottom>
```
needs conditional, otherwise it is always white (does not work in light mode)

same as the button